src/commands/flinkStatements.test.ts

```diff
@@ -0,0 +1,75 @@
+import assert from "assert";
+import sinon from "sinon";
+import * as vscode from "vscode";
+
+import { TextDocument } from "vscode-json-languageservice";
+import { createFlinkStatement } from "../../tests/unit/testResources/flinkStatement";
+import { FlinkStatementDocumentProvider } from "../documentProviders/flinkStatement";
+import { FlinkStatement } from "../models/flinkStatement";
+import { UriMetadataKeys } from "../storage/constants";
+import { ResourceManager } from "../storage/resourceManager";
+import { viewStatementSqlCommand } from "./flinkStatements";
+
+describe("commands/flinkStatements.ts", () => {
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("viewStatementSqlCommand", () => {
```

Comment:
New test suite over the 'view statement SQL document' command, low-hanging fruit.